u4-databases/views/uni-course-details.hbs

```diff
@@ -0,0 +1,6 @@
+{{#>uni iteration}}
```

Comment:
I think the `uni` include doesn't require any variables. You should delete `iteration` from this line.

Reply:
@WarojY - **ขออภัย! ฉันทำผิดพลาด / Apologies! I was mistaken.**

ฉันคิดว่านี่เป็น [Partial Parameter](https://handlebarsjs.com/guide/partials.html#partial-parameters) ที่ไม่ถูกต้อง แต่จริงๆ แล้วเป็น [Partial Context](https://handlebarsjs.com/guide/partials.html#partial-contexts) ที่ถูกต้อง
I thought this was an incorrect [Partial Parameter](https://handlebarsjs.com/guide/partials.html#partial-parameters), but it's actually a correct [Partial Context](https://handlebarsjs.com/guide/partials.html#partial-contexts).

ด้วยการเปลี่ยนบริบทเป็น `iteration` คุณสามารถเข้าถึงฟิลด์ของมันได้โดยตรง (เช่น: `iterationCode`) แทนที่จะเข้าถึงออบเจกต์จากนั้นจึงไปที่ฟิลด์ (เช่น: `iteration.iterationCode`) ฉันไม่ได้สอนสิ่งนี้ เพราะฉันไม่รู้ว่ามันมีอยู่จริง
By changing the context to `iteration`, you can directly access its fields (ex: `iterationCode`) instead of accessing the object then the field (ex: `iteration.iterationCode`). I didn't teach this, because I didn't know it existed.

ขอบคุณที่สอนฉันบางอย่าง :) / Thank you for teaching me something! :)